lambda/main.tf

```diff
@@ -68,7 +68,7 @@ resource "aws_cloudwatch_log_group" "this" {
 }
 
 resource "aws_cloudwatch_query_definition" "lambda_statistics" {
-  name = "Lambda Statistics - ${var.name}"
+  name = "Lambda Statistics / ${var.name}"
```

Comment:
This is not documented in Terraform (I didn't see it personally but discovered by accident) but putting a `/` character will treat this as a folder hierarchy.

Reply:
Very nifty - thanks for the PR!